config/prov/prov_test.yaml

```diff
@@ -175,6 +175,8 @@ k8s_cortx_deploy:
   run_s3bench_workload_flag: "True"
   destroy_setup_flag: "True"
   cortx_cluster_deploy_flag: "True"
+  deploy_ha_timeout_key: "CORTX_DEPLOY_HA_TIMEOUT"
+  deploy_ha_timeout_val: 1000s
```

Comment:
Is it necessary to specify the "s" suffix to indicate seconds?
Also, if the timeout value of 1000 is decided after discussion, could you please add comment/justification so that others can understand the intention of the value?

Reply:
yes `s` indicated seconds this is just time being workaround in code its mentioned in lib file, for bug 29667 its being modified.